src/OpenLoco/Ui/WindowManager.h

```diff
@@ -76,6 +76,22 @@ namespace OpenLoco::Ui::WindowManager
 
     uint8_t getVehiclePreviewRotationFrameUnk1();
     uint8_t getVehiclePreviewRotationFrameUnk2();
+
+    // 0x00523390
+    Ui::WindowNumber_t getToolWindowNumber();
+    void setToolWindowNumber(Ui::WindowNumber_t);
+
+    // 0x00523392
+    Ui::WindowType getToolWindowType();
+    void setToolWindowType(Ui::WindowType);
+
+    // 0x00523393
+    Ui::CursorId getCurrentToolCursor();
```

Comment:
this was moved from Ui.h - you could make an argument to leave it there but I opted to group these related bits of memory together.